data/rooms/RoomMessages.js

```diff
@@ -50,8 +44,17 @@ const AllRoomMessages = [
     chance: 1
   },
   {
-    regex: /\bth?a?n?[xk]s?q?\b/gim,
+    //tests: https://regex101.com/r/hH5cN7/42
+    regex: /(?:^|\s)(?:(?:th(?:n[qx]|x)|t[xyq])|than[kx](?:[sxz]){0,2}(?:[uq]|y(?:ou)?)?)\b/i,
     func: BotCommands.thanks
+  },
+  {
+    //tests: https://regex101.com/r/pT0zJ1/2
+    regex: /(?:[^@`]|\s+|^)(?:love|luv)\s?(?:u|you|me)?,?\s?(?:cbot|@?camperbot)\b/i,
```

Comment:
Document this new command clearly in the [**PR message**](https://github.com/FreeCodeCamp/camperbot/pull/78#issue-148545636).